multicluster/controllers/multicluster/internal/inbound_leader_manager.go

```diff
@@ -0,0 +1,85 @@
+package internal
+
+import (
+	"antrea.io/antrea/multicluster/controllers/multicluster/common"
+	"context"
+	"github.com/go-logr/logr"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+/**
+ *  This file contains implementation for the leader cluster to manage access to common area in the inbound mode
```

Comment:
I suggest to remove "inbound" and "inbound mode" from names and comments, until we decide to support "outbound" (and we should explain what does inbound/outbound mean somewhere), as it is confusing for readers have no context.